packaging/keybase.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
Re: @maxtaco's comment, I think that's an Ubuntu thing. They link `/bin/sh` to `/bin/dash` or something, rather than bash. Which they're allowed to do, though they've gotten a lot of complaints about it.